lib/code_corps/github/user.ex

```diff
@@ -50,6 +53,8 @@ defmodule CodeCorps.GitHub.User do
       Multi.new
       |> Multi.update(:user, changeset)
       |> Multi.run(:installations, fn %{user: %User{} = user} -> user |> associate_installations() end)
+      |> Multi.run(:tasks, fn %{user: %User{} = user} -> user |> associate_tasks() end)
+      |> Multi.run(:comments, fn %{user: %User{} = user} -> user |> associate_comments() end)
```

Comment:
I get why you chained more `Multi.run`s on here but I'm deeply concerned about the number of tasks/comments that we may be associating here. I had originally envisioned this process being fairly asynchronous and displaying to the user something to the effect of "your account is syncing with GitHub", even going so far as to be blocking.

This sync problem is going to be even larger when we sync large repos from GitHub to projects.

I know this adds some non-trivial complexity here. We could probably get away with this as a first pass, MVP and all. But this is something we should really circle back to, at the very least.

Reply:
Reading that these are `Repo.update_all` the operation may not be too slow. We should still write down an issue to benchmark certainly.

Syncing projects does seem like a different beast. That we need an issue for.